### PR TITLE
Organization address autocomplete

### PR DESCRIPTION
**Pull request by xuebingli**
@zendesk/volunteer-eng 

Address #4. Autocomplete address field in organization form.

* Extract `LocationField` out of `components/EventForm/index` to `lib/fields`
* Use `LocationField` in `OrganizationForm/index`
* Refactor unused imports

For future improvements, `DateField` can be extract out to `lib/fields` to avoid code duplication.

### CODE REVIEW

**Comment by xuebingli** [comment]
@zendesk/volunteer-eng ready for reviews

**Review by danielbreves — APPROVED** [review]

File: app/javascript/components/EventForm/index.js [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-import moment from 'moment-timezone'
 import R from 'ramda'
-import Geosuggest from 'react-geosuggest'
 import AutoComplete from 'material-ui/AutoComplete'
 import DatePicker from 'material-ui/DatePicker'
 import TimePicker from 'material-ui/TimePicker'
 
 import Callout from 'components/Callout'
-import Loading from 'components/LoadingIcon'
+import { LocationField } from 'lib/fields'
```
Comment (craig-day) [inline app/javascript/components/EventForm/index.js]: no components in `lib`. Those should only be helper functions that are truly shared in many places.

**Review by craig-day — CHANGES_REQUESTED** [review]
I don't want to get into the lib messiness. Please keep shared components in the components directory.

File: app/javascript/components/EventForm/index.js (hunk 1 above)
Comment (xuebingli) [reply app/javascript/components/EventForm/index.js]: Good point. I'll extract `LocationField` out as a separate component.

**Comment by xuebingli** [comment]
@craig-day 💇‍♂️

**Review by craig-day — APPROVED** [review]
Thanks!

**Pull request merged**